adColony/android/src/com/submarine/adcolony/AdColonyAndroidNetwork.java

```diff
@@ -0,0 +1,91 @@
+package com.submarine.adcolony;
+
+import com.badlogic.gdx.backends.android.AndroidApplication;
+import com.jirbo.adcolony.*;
+
+/**
+ * Created by mariam on 3/9/15.
+ */
+public class AdColonyAndroidNetwork implements AdColonyNetwork {
+
+    private final AndroidApplication androidApplication;
+    private String clientOptions;
+    private String appId;
+    private String[] zoneIds;
+    private AdColonyHandler handler;
+
+
+    public AdColonyAndroidNetwork(AndroidApplication androidApplication, String clientOptions, String appId, String[] zoneIds, AdColonyHandler handler) {
+        this.androidApplication = androidApplication;
+        this.clientOptions = clientOptions;
+        this.appId = appId;
+        this.zoneIds = zoneIds;
+        this.handler = handler;
+
+        configureAdColony();
+    }
+
+    private void configureAdColony() {
+        AdColony.configure(androidApplication, clientOptions, appId, zoneIds);
+
+
+        AdColonyV4VCListener listener = new AdColonyV4VCListener() {
+            public void onAdColonyV4VCReward(AdColonyV4VCReward reward) {
+                if (reward.success()) {
+                    handler.reward();
+                }
+            }
+        };
+        AdColony.addV4VCListener(listener);
+    }
+
```

Comment:
I think just pause will be beater then pauseAdColony and resume then resumeAdColony. D